### PR TITLE
Tool for bulk transcription

### PR DESCRIPTION
**Pull request by tilmankamp**

### CODE REVIEW

File: transcribe.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                    break
+                decoded = ctc_beam_search_decoder_batch(batch_logits, batch_lengths, Config.alphabet, FLAGS.beam_width,
+                                                        num_processes=num_processes,
+                                                        scorer=scorer)
+                decoded = list(d[0][1] for d in decoded)
+                transcripts.extend(zip(starts, ends, decoded))
+                bar.update(len(transcripts))
+            bar.finish()
+            transcripts.sort(key=lambda t: t[0])
+            transcripts = list(map(lambda t: {'start': int(t[0]), 'end': int(t[1]), 'transcript': t[2]}, transcripts))
```
Comment (reuben) [inline transcribe.py]: How about a comprehension instead?

```python
transcripts = [{'start': int(start), 'end': int(end), 'transcript': transcript} for start, end, transcript in transcripts]
```

File: transcribe.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        sys.exit(1)
+    if src_path.endswith('.catalog'):
+        if FLAGS.dst:
+            log_error('Parameter --dst not supported if --src points to a catalog')
+            sys.exit(1)
+        catalog_dir = os.path.dirname(src_path)
+        with open(src_path, 'r') as catalog_file:
+            catalog_entries = json.load(catalog_file)
+        catalog_entries = list(map(lambda e: (resolve(catalog_dir, e['audio']), resolve(catalog_dir, e['tlog'])),
+                                   catalog_entries))
```
Comment (reuben) [inline transcribe.py]: Same here?
```python
catalog_entries = [(resolve(catalog_dir, e['audio']), resolve(catalog_dir, e['tlog'])) for e in catalog_entries]
```

File: util/audio.py [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        return self.open_file
+
+    def __exit__(self, *args):
+        if not self.as_path:
+            self.open_file.close()
+        if self.tmp_file_path is not None:
+            os.remove(self.tmp_file_path)
+
+
+def load_audio(audio_path, audio_format=DEFAULT_FORMAT):
```
[the remark this answers is not in the document]
Comment (tilmankamp) [reply util/audio.py]: Removed it

File: util/audio.py [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+            if num_unvoiced > threshold * ring_buffer.maxlen:
+                triggered = False
+                yield b''.join(voiced_frames), \
+                      frame_duration_ms * max(0, frame_index - len(voiced_frames)), \
+                      frame_duration_ms * frame_index
+                ring_buffer.clear()
+                voiced_frames = []
+        frame_index += 1
+    if triggered:
+        pass
```
[the remark this answers is not in the document]
Comment (tilmankamp) [reply util/audio.py]: Yes - removed it.

**Comment by tilmankamp** [comment]
@reuben Could  you have a second look (review)? I addressed all your comments, but there are some important changes in `transcribe.py` to prevent memory leaks...

**Pull request merged**